tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -203,7 +219,23 @@ install_dragonball_experimental_kernel() {
 		&& return 0
 
 	info "kernel version ${kernel_version}"
-	DESTDIR="${destdir}" PREFIX="${prefix}" "${kernel_builder}" -e -t dragonball -v ${kernel_version}	
+	DESTDIR="${destdir}" PREFIX="${prefix}" "${kernel_builder}" -e -t dragonball -v ${kernel_version}
+}
+
+#Install GPU and SNP enabled kernel asset
+install_kernel_gpu_snp() {
+	install_kernel_helper \
+		"assets.kernel.snp.version" \
+		"kernel-gpu-snp" \
+		"-x snp -g nvidia"
+}
+
+#Install GPU and TDX enabled kernel asset
+install_kernel_gpu_tdx() {
+	install_kernel_helper \
+		"assets.kernel-tdx-experimental.version" \
+		"kernel-gpu-tdx" \
+		"-x tdx -g nvidia"
```

Comment:
You'll need something like:
```
	local kernel_url="$(get_from_kata_deps assets.kernel-tdx-experimental.url)"

	install_kernel_helper \
		"assets.kernel-tdx-experimental.version" \
		"kernel-tdx-experimental" \
		"-x tdx -u ${kernel_url}"
```

As we pass the `${kernel_url}` here, yq won't be used from inside the nested container.